examples/flow.cpp

```diff
@@ -251,15 +251,31 @@ try
     const PhaseUsage pu = Opm::phaseUsageFromDeck(deck);
     Opm::BlackoilOutputWriter outputWriter(grid, param, eclipseState, pu );
 
+    int numCells = Opm::UgGridHelpers::numCells(grid);
```

Comment:
the problem with this is that something like the UgGridHelpers would need to be moved to opm-material and I'd rather avoid that...